tests/MongoDB.Driver.Core.Tests/Core/Operations/BulkMixedWriteOperationTests.cs

```diff
@@ -1112,6 +1114,7 @@ public void WriteConcern_should_work()
             var subject = new BulkMixedWriteOperation(_collectionNamespace, requests, _messageEncoderSettings)
             {
                 IsOrdered = false,
+                RetryRequested = retryRequested,
```

Comment:
Because we never tested with RetryRequested=true we didn't catch this bug before.